chromium_src/chrome/browser/ui/views/frame/multi_contents_view_mini_toolbar.cc

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/ui/views/frame/multi_contents_view_mini_toolbar.h"
+
+#include <memory>
+#include <optional>
+
+#include "chrome/app/vector_icons/vector_icons.h"
+#include "chrome/browser/ui/tabs/split_tab_menu_model.h"
+#include "ui/menus/simple_menu_model.h"
+
+// defined at brave_multi_contents_view_mini_toolbar.cc to avoid
+// brave/components/vector_icons dependency here.
+const gfx::VectorIcon& GetMoreVerticalIcon();
+
+// defined at brave_split_tab_menu_model.cc to avoid BraveSplitTabMenuModel
+// dependency here.
+std::unique_ptr<ui::SimpleMenuModel> CreateBraveSplitTabMenuModel(
```

Comment:
thinking out loud: I'm not so sure about this approach, where we're declaring global function in .cc file without having declaration in a proper header. Could it be better to move these into header and inside class and use compile time polymorphism? Maybe it could save us from pollute global namespace.

Reply:
Good suggestion. this method can be handled in that way.

One limition of this method is - when that method is in ctor. I can't use it for `GetMoreVerticalIcon()`.

Reply:
good idea. using static method seems better. changed both. 👍🏼